Formula/flex.rb

```diff
@@ -34,19 +35,30 @@ class Flex < Formula
   uses_from_macos "bison" => :build
   uses_from_macos "m4"
 
-  def install
-    if build.head?
-      ENV.prepend_path "PATH", Formula["gnu-sed"].opt_libexec/"gnubin"
-
-      system "./autogen.sh"
-    end
+  resource "bootstrap" do
+    url "https://github.com/westes/flex/releases/download/v2.6.4/flex-2.6.4.tar.gz"
+    sha256 "e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995"
+  end
```

Comment:
I don't think we want to carry around a head-only resource that is identical to the stable tarball. This makes version updates, for example, needlessly complicated.

We should just do `depends_on :macos` in the `head` block. All this extra stuff is needed only on Linux because macOS already provides `flex`.

Reply:
That's not really complicated and does not add problems that I could see.

Also, the last release of flex was in 2017, so I guess there is not really any need to do a new stable release of flex ever.

Also, I think it's not even needed to even provide a build of flex master, so I think it would be fine to even remove head completely.

But I am also OK with just doing depends_on :macos in the head block as you say. pushed.